docs/monitoring.md

```diff
@@ -307,6 +307,7 @@ API path listed as below:
 | /applications         | List all running application's ids of the cluster.                                                                                  |
 | /shuffles             | List all running shuffle keys of the service. It will return all running shuffle's key of the cluster.                              |
 | /listTopDiskUsedApps  | List the top disk usage application ids. It will return the top disk usage application ids for the cluster.                         |
+| /exclude?id=${ID}     | Exclude the worker given worker id. This worker is added into excluded workers manually.                                            |
```

Comment:
Support excluding multiple workers in one request? One is just a special case

Reply:
Support excluding multiple workers sounds reasonable.